src/Stratis.Bitcoin.Features.PoA/PoAConsensusOptions.cs

```diff
@@ -6,14 +6,14 @@ namespace Stratis.Bitcoin.Features.PoA
 {
     public class PoAConsensusOptions : ConsensusOptions
     {
-        /// <summary>Public keys of all federation members at the start of the chain.</summary>
+        /// <summary>Public keys and other federation members related information at the start of the chain.</summary>
         /// <remarks>
         /// Do not use this list anywhere except for at the initialization of the chain.
         /// Actual collection of the federation members can be changed with time.
-        /// Use <see cref="FederationManager.GetFederationMembers"/> as a source of
+        /// Use <see cref="IFederationManager.GetFederationMembers"/> as a source of
         /// up to date federation keys.
         /// </remarks>
-        public List<PubKey> GenesisFederationPublicKeys { get; protected set; }
+        public List<IFederationMember> GenesisFederation { get; protected set; }
```

Comment:
I'm thinking we should rename this `GenesisFederationMembers`. `GenesisFederation` doesn't really imply that its a list of members?

Reply:
Fixed